### ISSUE TITLE
add-env gave a null pointer with env name "k8s-test"

### ISSUE THREAD

**Issue by vinok88** [wso2/product-apim-tooling#582]
**Description:**
Following error was given with add-env command with environment name "k8s-test". Worked fine with name "k8s"

```
[A494823@TTGSSDV0VRHAP52 scenario-24]$ /home/A494823/istio/apictl/apictl import-api -f petstore-api/ -e k8s-test -k

panic: runtime error: invalid memory address or nil pointer dereference

[signal SIGSEGV: segmentation violation code=0x1 addr=0x88 pc=0xccf738]

 

goroutine 1 [running]:

github.com/wso2/product-apim-tooling/import-export-cli/impl.injectParamsToAPI(0xc000381220, 0x1f, 0xc0000425a0, 0x5d, 0x7ffcbd6f4730, 0x8, 0x1, 0x0, 0x0)

        /home/naduni/Documents/Wso2/CLI/product-apim-tooling/import-export-cli/impl/importAPI.go:647 +0x198

github.com/wso2/product-apim-tooling/import-export-cli/impl.ImportAPI(0xc0005a61b0, 0x24, 0xc00032a580, 0x38, 0x7ffcbd6f4730, 0x8, 0x7ffcbd6f471f, 0xd, 0xecea6e, 0xf, ...)

        /home/naduni/Documents/Wso2/CLI/product-apim-tooling/import-export-cli/impl/importAPI.go:978 +0x1121

github.com/wso2/product-apim-tooling/import-export-cli/impl.ImportAPIToEnv(0xc0005a61b0, 0x24, 0x7ffcbd6f4730, 0x8, 0x7ffcbd6f471f, 0xd, 0xecea6e, 0xf, 0x7ffcbd000100, 0x8, ...)

        /home/naduni/Documents/Wso2/CLI/product-apim-tooling/import-export-cli/impl/importAPI.go:927 +0xf8

github.com/wso2/product-apim-tooling/import-export-cli/cmd.glob..func20(0x16e69e0, 0xc00010aeb0, 0x0, 0x5)

        /home/naduni/Documents/Wso2/CLI/product-apim-tooling/import-export-cli/cmd/importAPI.go:67 +0x1c2

github.com/spf13/cobra.(*Command).execute(0x16e69e0, 0xc00010adc0, 0x5, 0x5, 0x16e69e0, 0xc00010adc0)

        /home/naduni/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830 +0x29d

github.com/spf13/cobra.(*Command).ExecuteC(0x16e8a60, 0x443aba, 0x16892c0, 0xc000000300)

        /home/naduni/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914 +0x2fb

github.com/spf13/cobra.(*Command).Execute(...)

        /home/naduni/go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864

github.com/wso2/product-apim-tooling/import-export-cli/cmd.Execute()

        /home/naduni/Documents/Wso2/CLI/product-apim-tooling/import-export-cli/cmd/root.go:79 +0x31

main.main()

        /home/naduni/Documents/Wso2/CLI/product-apim-tooling/import-export-cli/apictl.go:24 +0x20
```

**Suggested Labels:**

**Suggested Assignees:**

**Affected Product Version:**
Version: 3.2.1
**OS, DB, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

**Comment by npamudika** [comment]
Hi @vinok88, 

Can you please provide the command you tried to add the environment k8s-test? Did you get the error when adding the environment or importing an API to the k8s-test environment as in the provided logs I can see an import command?

Thanks,
Naduni

**Comment by vinok88** [comment]
As you can see the command is there at the top of the error log. It is when importing an API.

Thanks,
Vinod

**Comment by uvindra** [comment]
This issue is not reproducible in the latest 4.0.0 version of the ctl
```
./apictl import api -f k8s-test/PizzaShackAPI_1.0.0.zip -e k8s-test
Successfully imported API.
```

This nil pointer deref has already been fixed in the 3.2.x branch and will be available when sending the next patch release v3.2.2
